devices/xiaomi.js

```diff
@@ -98,12 +99,24 @@ const fzLocal = {
                 case 0x0275:
                     result['valve_alarm'] = {1: true, 0: false}[value];
                     break;
+                case 0x00f7:
+                    result['valve_alarm'] ={1: true, 0: false}[value.readUInt32LE(36)];
+                    result['occupied_heating_setpoint'] = value.readUint16LE(32)/100;
+                    result['local_temperature'] = value.readUint16LE(28)/100;
+                    break;
+                case 0x027d:
+                    result['schedule'] = {1: 'ON', 0: 'OFF'}[value];
+                    break;
+                case 0x0276: {
+                    const schedule = xiaomiUtils.readSchedule(value);
+                    result['schedule_settings_json'] = schedule;
```

Comment:
Why expose this twice? My preference would be to drop `schedule_settings_json`

Reply:
Compared to most settings, this has quite a complicated format. Therefore, `schedule_settings_json` is meant for programmatic use and power users, while `schedule_settings` is a simplified alternative for simple management through the frontend. What speaks against providing both?

Reply:
I prefer the `schedule_settings_json` over the `schedule_settings` (drop `schedule_settings` and rename `schedule_settings_json` to `schedule_settings`).

If it is exposed correctly (https://github.com/Koenkk/zigbee-herdsman-converters/pull/5058/files#r1037429975) then it will work for all users.

Reply:
That would be great and my preferred way too, but as I commented below, I am lacking the guidance to implement it like this.